test/e2e/test-e2e-suite-template.yml

```
@@ -0,0 +1,73 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: osde2e-focused-tests
+
+parameters:
+  - name: OSDE2E_CONFIGS
+    required: true
+  - name: TEST_E2E_SUITE_IMAGE
+    required: true
+  - name: OCM_TOKEN
+    required: true
+  - name: OCM_CCS
+    required: false
+  - name: AWS_ACCESS_KEY_ID
+    required: false
+  - name: AWS_SECRET_ACCESS_KEY
+    required: false
+  - name: CLOUD_PROVIDER_REGION
+    required: false
+  - name: GCP_CREDS_JSON
+    required: false
+  - name: JOBID
+    generate: expression
+    from: "[0-9a-z]{7}"
+  - name: IMAGE_TAG
+    value: ''
+    required: true
+  - name: LOG_BUCKET
+    value: 'osde2e-logs'
+objects:
+  - apiVersion: batch/v1
+    kind: Job
+    metadata:
+      name: configuration-anomaly-detection-${IMAGE_TAG}-${JOBID}
+    spec:
+      backoffLimit: 0
+      template:
+        spec:
+          restartPolicy: Never
+          containers:
+            - name: osde2e
+              image: quay.io/redhat-services-prod/osde2e-cicada-tenant/osde2e:latest
+              command:
+                - /osde2e
+              args:
+                - test
+                - --configs
+                - ${OSDE2E_CONFIGS}
+              securityContext:
+                runAsNonRoot: true
+                allowPrivilegeEscalation: false
+                capabilities:
+                  drop: ["ALL"]
+                seccompProfile:
+                  type: RuntimeDefault
+              env:
+                - name: TEST_E2E_SUITEES
```

```suggestion
                - name: TEST_HARNESSES
```
this one can't swap yet, osde2e is looking for the env named this. I'll swap this here when we change things